asv_bench/benchmarks/benchmarks.py

```diff
@@ -992,4 +993,96 @@ def time_float_32(self, shape, method):
         execute(self.df_ts_float32.fillna(method=method))
 
 
+class TimeLevelAlign:
+    params = [get_benchmark_shapes("TimeLevelAlign")]
```

Comment:
It's probably not a good idea to use the forms for the unary operation that supposed to be used for the binary operations.

Reply:
As Align  aligns two objects. I was thinking this would be a binary operation.

Reply:
oh, you are right! ok